scripts/release.sh

```diff
@@ -244,6 +282,21 @@ function parse_flags() {
     shift
   done
 
+  # Do auto release unless release is forced
+  if (( is_auto_release )); then
+
+    if (( is_dot_release )); then
+      abort "error: cannot have both --dot-release and --auto-release set simultaneously"
```

Comment:
`abort()` already displays "error: " at the beginning of the message, here and everywhere.